src/client/utils/property-overriding.ts

```diff
@@ -1,4 +1,5 @@
 import nativeMethods from '../sandbox/native-methods';
+import INTERNAL_PROPS from '../../processing/dom/internal-properties';
```

Comment:
Rename the
`src/client/utils/property-overriding.ts` file to
to `src/client/utils/overriding.ts` since now this file contains functions for override not only properties.